pyhf/utils.py

```diff
@@ -127,27 +127,34 @@ def pvals_from_teststat(sqrtqmu_v, sqrtqmuA_v):
     return CLsb, CLb, CLs
 
 
-def runOnePoint(muTest, data, pdf, init_pars=None, par_bounds=None):
+def cls(mu_test, data, pdf, init_pars=None, par_bounds=None, **kwargs):
```

Comment:
I'd maybe call this `pyhf.utils.hypotest`

Reply:
Changed in https://github.com/diana-hep/pyhf/pull/341/commits/490b22da1b8d2c6849523dcbfe340a211ac4dfd5